bundles/net.tourbook/src/net/tourbook/tag/Dialog_TourTag.java

```diff
@@ -124,24 +159,68 @@ private void createUI(final Composite parent) {
             GridDataFactory.fillDefaults().align(SWT.FILL, SWT.CENTER).applyTo(label);
 
             _txtName = new Text(container, SWT.BORDER);
-            GridDataFactory.fillDefaults().grab(true, false).applyTo(_txtName);
+            GridDataFactory.fillDefaults().span(3, 1).grab(true, false).applyTo(_txtName);
          }
          {
-            // Text: Notes
+            // Text: Image File Path
+            final Label label = UI.createLabel(container, UI.EMPTY_STRING);
+            label.setText(Messages.Dialog_TourTag_Label_Image);
+            GridDataFactory.fillDefaults().align(SWT.FILL, SWT.BEGINNING).applyTo(label);
 
+            _canvasTagImage = new Label(container, SWT.TRANSPARENCY_ALPHA);
```

Comment:
I can't reproduce the issue locally. Are you sure that your image original is not horizontal and that maybe Windows Explorer is rotating it in the windows dialog ?
Also, [my code ](https://github.com/FJBDev/mytourbook/blob/6c7e846fac19ef4a4788f6c39b96df4277f0743a/bundles/net.tourbook.photo/src/net/tourbook/photo/ImageUtils.java#L332)never rotates any image

![image](https://user-images.githubusercontent.com/25235665/218577894-4bc9e5d6-6bb0-48f4-82cf-fd4de18d8c06.png)

Reply:
Do you respect the image orientation?

![image](https://user-images.githubusercontent.com/1283445/218657312-c5e442af-f315-421c-b357-150418b8f3f9.png)

In the MT photo feature, photos are rotated according to the orientation

![image](https://user-images.githubusercontent.com/1283445/218657466-cdd9a5fa-2cfd-418f-bab4-fd8bf4c4c7a3.png)

Reply:
> Do you respect the image orientation?

I believe so yes because I keep the orientation unchanged, at least that's what I see [in my code.](https://github.com/FJBDev/mytourbook/blob/6c7e846fac19ef4a4788f6c39b96df4277f0743a/bundles/net.tourbook/src/net/tourbook/ui/UI.java#L890)
Should I not be respecting the orientation and decide to rotate the image if needed ?

Reply:
My original message "The tag image is rotated" should be the opposite: "The tag image is NOT rotated". 

In your [code](https://github.com/FJBDev/mytourbook/blob/6c7e846fac19ef4a4788f6c39b96df4277f0743a/bundles/net.tourbook/src/net/tourbook/ui/UI.java#L890) I don't see that you read the EXIF orientation (when available) and rotate the image accordingly.

My suggestion

* When an image do not contain EXIF orientation data, do not rotate the image
* When an image do contain EXIF orientation data, then rotate the image accordingly

Reply:
Ok, that makes more sense now, I will implement that

Reply:
Fixed 

![image](https://user-images.githubusercontent.com/25235665/219716790-5ad8b31d-cf5f-413a-b96b-0629e0bb25c8.png)

Reply:
I still see the image as not rotated

![image](https://user-images.githubusercontent.com/1283445/219848282-a47d2407-2e6c-4abd-8d2d-59e518247ee2.png)

The photo gallery is rotating it correctly

![image](https://user-images.githubusercontent.com/1283445/219848224-f853ebf2-254b-48d1-a7be-a8f981de6587.png)

Reply:
In your code...

![image](https://user-images.githubusercontent.com/1283445/219851100-bc5182d4-9c3c-47bb-852e-5833073064e7.png)

...`rotation` could also be `< 0` but when fixing this, the image is still not rotated as it should. 

Maybe you should use the already implemented image resize/rotation code.

Reply:
>Maybe you should use the already implemented image resize/rotation code.

I thought of it but it didn't seem that the `photo` code was keeping the image transparency when resizing an image so that's why i wrote new code.

Reply:
Actually, I reused the already implemented image resize/rotation code and it has fixed the below issues:
- Image rotation
- Image pixelization


The image rotation should be fixed now
![image](https://user-images.githubusercontent.com/25235665/220474240-a0b72a21-4220-4e94-9b12-563c4fc9db93.png)

Reply:
![image](https://user-images.githubusercontent.com/1283445/220536889-15e7aa96-2c98-40b9-bdc1-727e3c5d10e1.png)

😄

Reply:
Loading larger photo images are now also much faster